setup.py

```diff
@@ -33,7 +33,7 @@
 packages = ["apiclient", "googleapiclient", "googleapiclient/discovery_cache"]
 
 install_requires = [
-    "httplib2>=0.17.0,<1dev",
+    "httplib2>=0.9.2,<1dev",
```

Comment:
should we add a comment on this line so folks don't mistakenly try to update it?

Reply:
Done.